pkg/apis/devices/v1beta1/device_instance_types.go

```diff
@@ -0,0 +1,219 @@
+/*
+Copyright 2023 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"encoding/json"
+	v1 "k8s.io/api/core/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// DeviceSpec represents a single device instance.
+type DeviceSpec struct {
+	// Required: DeviceModelRef is reference to the device model used as a template
+	// to create the device instance.
+	DeviceModelRef *v1.LocalObjectReference `json:"deviceModelRef,omitempty"`
+	// NodeName is a request to schedule this device onto a specific node. If it is non-empty,
+	// the scheduler simply schedules this device onto that node, assuming that it fits
+	// resource requirements.
+	// +optional
+	NodeName string `json:"nodeName,omitempty"`
```

Comment:
Is it needed for node selector fo node group?

Reply:
No need, now one device instance will match one device in a node.

Reply:
yes, we use `NodeName ` at here.